Rakefile

```diff
@@ -21,22 +21,20 @@ namespace :db do
   end
 end
 
-task 'db/Category-Emoji.json' do |t|
-  system 'plutil', '-convert', 'json', '-r',
-    '/System/Library/Input Methods/CharacterPalette.app/Contents/Resources/Category-Emoji.plist',
-    '-o', t.name
+file 'db/raw-emoji.json' do |t|
+  system 'curl', '-fsSL', 'https://raw.githack.com/koddsson/emoji.json/11.0.0/emoji.json', '-o', t.name
```

Comment:
Haha I totally don't remember why I put this through GitHack. I'll change it 😄 

`koddsson/emoji.json` is a fork of `amio/emoji.json` that has a 11.0.0 branch that includes the category I added to the project in https://github.com/amio/emoji.json/pull/8. The 11.0.0 tag in `amio/emoji.json` doesn't have the category field. I was hoping we could use the fork here and then switch to just use the `amio/emoji.json` project to generate emojis once Emoji 12.0.0 is supported in more platforms.

Currently `amio/emoji.json` parses the Unicode Consortium web page with [this Chrome script](https://github.com/amio/emoji.json/blob/master/tools/gen-json.js). It's a pretty manual process and I did try parsing the page with NodeJS + Cheerio but every time I would load the document I would get memory exceptions (presumable since the HTML document is so large). I can take another stab at bringing a script into the repo.